arch/arm/arm64/include/uk/asm/lcpu.h

```diff
@@ -51,32 +49,25 @@ struct __regs {
 	/* Link Register (x30) */
 	unsigned long lr;
 
-	/* Exception Link Register */
-	unsigned long elr_el1;
+	/* Program Counter */
```

Comment:
I was initially confused by this as in AArch64 there is no way to get access to the PC. Of course ELR does contain a previous value of the PC, yet I think it would be more precise if this was called elr_el1. If you think I'm being too pedantic, feel free to ignore this comment though ;)